CHANGELOG.rst

```diff
@@ -4,6 +4,13 @@
 - Nothing changed yet.
 
 
+(2019-04-06)
```

Comment:
Dates are for when releases are made. Just add your changelog in place of "- Nothing changed yet."